src/jit/decomposelongs.cpp

```diff
@@ -1013,15 +1013,25 @@ GenTree* DecomposeLongs::DecomposeShift(LIR::Use& use)
 {
     assert(use.IsInitialized());
 
-    GenTree* tree      = use.Def();
-    GenTree* gtLong    = tree->gtGetOp1();
+    GenTree* shift     = use.Def();
```

Comment:
> But this should have been done separately, a lot of changes in this diff are caused by this and not by the actual bug fix.

Yes, you're right. In this case I elected to fold the changes together b/c it helped me reason about the change, but I should have undone the renames before submitting the PR.